docs/introduction/resources.md

```diff
@@ -68,6 +68,15 @@ Here are some examples from manufacturing situations:
 
 Note in the last case, a transfer of rights means a different resource. This is common with resources that are not serialized, where one logical resource has a quantity greater than 1, and the individual instances are substitutable. (Think nuts and bolts, grain, strawberries, bottles of beer in cases, etc.)
 
+##### Stage and state
+
+Sometimes part of the logical identification of a resource includes:
+* stage: the ProcessSpecification of the most recent process the resource was output from, with the `modify` action
+* state: the Action (`pass` or `fail` only) of the most recent economic event the resource was output from
+
+The stage is used when a resource passes through multiple stages in its lifetime, and that is important when determining the next process it should be input for.  For example, in creating a translation, you might have one translated document pass through translation, editing, proofreading, formatting stages. You don't want to bring that resource into the formatting stage until it has been proofread, for example. Or you might have a testing stage for a component or product, in which case you don't want to consume or transfer the resource until it both has been through the testing stage, and had a `pass` output result.
+
+These can be defined on the recipe or the plan, showing where an input flow expects a certain stage and/or state.  In the observation layer, the input event should then check those as required when offering selections of possible input resources.
```

Comment:
Suggested change:
>  In the observation layer, the input event should then check those as required when offering selections of possible input resources.

Change to:
In user-interface forms for adding EconomicEvents to fulfill Commitments, the input event form should query EconomicResources for required stage and state when offering selections of possible input resources.

Reply:
Changed except for the "to fulfill Commitments", since that may or may not be the case.